src/sentry/utils/raven.py

```diff
@@ -34,16 +35,26 @@ def is_enabled(self):
             return False
         return settings.SENTRY_PROJECT is not None
 
-    def capture(self, *args, **kwargs):
```

Comment:
this will cause it to do a lot of excess work in systems where there is no remote